### DISCUSSION TITLE
Gif/Video inserter

### DISCUSSION THREAD

**Discussion by Admiralgogo** [ScratchAddons/ScratchAddons#6908] [Ideas]
add a thing so you can insert videos/gifs into sprites (with no sound), and can choose which frams you want to delete if more

**Comment by CubesterYT** [comment]
Unless you mean it without choosing frames, then it may be addable.

**Reply by Admiralgogo** [reply]
ok

**Comment by BroJac5246** [comment]
I would definitely use this. We could even offer custom blocks for running them at the correct frame rate.
